packages/ui/src/components/Form/customField/CustomSelectField.tsx

```diff
@@ -0,0 +1,6 @@
+import { SelectField, SelectFieldProps } from '@kaoto-next/uniforms-patternfly';
+import { FunctionComponent } from 'react';
+
+export const CustomSelectField: FunctionComponent<SelectFieldProps> = (props) => {
+  return <SelectField {...(props as SelectFieldProps)} placeholder="Please select an option..." />;
+};
```

Comment:
wrapper for the `CustomSelectField` so we could add an additional property `placeholder`